Gopkg.toml

```diff
@@ -93,11 +94,20 @@
 
 [[override]]
   name = "k8s.io/apiextensions-apiserver"
-  revision = "05e89e265cc594459a3d33a63e779d94e6614c63"
+  version = "kubernetes-1.13.6"
+
+ [[override]]
+   name = "k8s.io/apiserver"
+   version = "kubernetes-1.13.6"
+
+ [[override]]
+   name = "k8s.io/api"
+   version = "kubernetes-1.13.6"
 
 [[override]]
   name = "k8s.io/kubernetes"
-  revision = "54a352dda957bce0f88e49b65a6ee8bba8c0ba74"
+  version = "v1.13.6"
+  # revision = "721bfa751924da8d1680787490c54b9179b1fed0"
```

Comment:
I'm surprised you had to bump the version. Did you import a new variable or pkg?

Reply:
And I'm sorry you had to go through this. The package dependency war has scarred Kubernetes development, with `client-go` and `k8s` using `godep` and `helm` using `glide`, making it impossible to use `dep` or `gomodules` with them :(

I appreciate your fight through!

Reply:
I wanted a newer version of terratest (`0.13.30`) that has `k8s.GetRole()`. From there I just muddled my way through the dependency errors until I found a winning combination. This may not be the only (or best) configuration...